meta-mender-core/recipes-mender/mender/mender_2.1.1.bb

```diff
@@ -0,0 +1,28 @@
+require mender.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender;protocol=https;branch=2.1.x"
+
+# Tag: 2.1.1
```

Comment:
b1, right?

I know we don't usually do it for patch releases, but it's hard to change the pattern when the rest of the release is beta.

Reply:
Discussed in person.

The tag for Mender client is 2.1.1 (not beta). Agreed in that it is inconsistent with a beta release, but it has been properly tested and the risk is very low.